tools/wt_verify.py

```diff
@@ -0,0 +1,157 @@
+    """
+    Execute a given command and return its output.
+    """
+    try:
+        result = subprocess.run(command, shell=True, check=True,
+                                stdout=subprocess.PIPE,
+                                universal_newlines=True)
+    except subprocess.CalledProcessError as e:
+        print(f"Error executing command: {e}", file=sys.stderr)
+        sys.exit(1)
+    
+    return result.stdout
+
+
+def find_wt_exec_path():
+    """
+    Find the path of the WT tool executable. 
+    
+    We expect to find exactly one wt binary. Otherwise exit and prompt the user to provide an explicit path to the binary
+    """
+    path = f"{os.path.dirname(os.path.abspath(__file__))}/../."
+
+    try:
+        result = subprocess.run(['find', path, '-maxdepth', '2', '-name', 'wt'],
+                                stdout=subprocess.PIPE, text=True)
+    except subprocess.CalledProcessError as e:
+        print(f"Error executing command: {e}", file=sys.stderr)
+        sys.exit(1)
+
+    list_of_paths = result.stdout.split('\n')
+    # Sanity check to remove empty new lines from find output
+    result = [path for path in list_of_paths if path != '']
+
+    if len(result) > 1:
+        print("Error: multiple wt executables found. Please provide wt executable path using the -wt flag:")
+        for path in result:
+            print(path)
+        exit(1)
+    if len(result) == 0:
+        print("Error: wt executable not found. Please provide path using the -wt flag.")
+        exit(1)
+
+    return result[0]
+
+
+def construct_command(args):
+    """
+    Construct the WiredTiger verify command based on provided arguments.
+    """
+    if args.wt_exec_path:
+        command = f"{args.wt_exec_path}"
+    else:
+        command = f"{find_wt_exec_path()}"
+
+    command += f" -h {args.home_dir} verify -t"
+    if args.dump:
+        command += f" -d {args.dump}"
+    if args.file_name:
+        command += f" \"{args.file_name}\""
+    return command
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Script to run the WiredTiger verify command with specified options.")
+    parser.add_argument('-hd', '--home_dir', default='.', help='Path to the WiredTiger database home directory (default is current directory).')
+    parser.add_argument('-f', '--file_name', required=True, help='Name of the WiredTiger file to verify.')
+    parser.add_argument('-wt', '--wt_exec_path', help='Path of the WT tool executable.')
+    parser.add_argument('-o', '--output_file', help='Option to save output in given output file.')
+    parser.add_argument('-d', '--dump', required=True, choices=['dump_pages'], help='Option to specify dump_pages or dump_blocks configuration.')
+    parser.add_argument('-p', '--print_output', action='store_true', default=False, help='Print the output (default is on)')
+    parser.add_argument('-v', '--visualize', choices=['page_sizes', 'entries', 'dsk_image_sizes'], help='Type of visualization')
```

Comment:
This is for a later PR, but it'd be good to enable multiple choices at the same time (i.e. `--visualise=[page_sizes,entries]`)

Reply:
It seems that by simply giving `args='+' to my parser I can allow user to give 1 or more arguments. Thank to this interesting [stackoverflow](https://stackoverflow.com/questions/15753701/how-can-i-pass-a-list-as-a-command-line-argument-with-argparse) page - quite a tiny change so I've added it for extensibility. e750f0f2514188309d6f9c8a6d18f4db768fd146